app/routes/core.rb

```diff
@@ -21,7 +21,7 @@ class Core < Sinatra::Application
         enable :show_exceptions
       end
 
-      unless settings.test?
+      unless settings.test? || settings.development?
```

Comment:
You can try it yourself by deleting the conditional and `enable`/`disable`ing `:show_exceptions` on line 21. (It doesn't require a reboot.)